projects/storefrontlib/src/lib/ui/layout/main/main.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
What is this file for? + there is no html or css for the templateUrl and StyleUrls.

Reply:
delete it, idk why some old files i already deleted appeared after merge